index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css"
+     integrity="sha384-rwoIResjU2yc3z8GV/NPeZWAv56rSmLldC3R/AZzGRnGxQQKnKkoFVhFQhNUwEyJ" crossorigin="anonymous">
+    <link rel="stylesheet" href="style.css">
+    <title>Weather App</title>
+  </head>
+
+  <body>
+    <div class="container top main">
+      <h1 class="text-center title">Current Weather</h1>
+        <div class="row justify-content-center top">
+            <div class="form-group row justify-content-center">
+              <div class="col-auto">
+                <input type="text" class="user-input" placeholder="Enter City Name">
+              </div>
+            </div>
+        </div>
+        <div class="form-group row justify-content-center">
+          <div class="col-auto">
+            <input type="button" class="btn-primary search" value="Search">
+          </div>
+          <div class="col-auto">
+            <p></p>
+          </div>
+          <div class="col-auto">
+            <input type="button" class="btn-primary location" value="Use location">
+          </div>
+        </div>
+
+        <div class="row display justify-content-center top">
```

Comment:
So `display` is for the current weather, and `five-day` is for the 5-day forecast? I'd rename `display`.